src/i18n/en.json

```diff
@@ -1466,5 +1466,19 @@
     "dashboard.agreements.widget.title": "Agreements",
     "dashboard.agreements.widget.noData": "There are no agreements to display.",
     "dashboard.agreements.widget.createObjective": "Create a new agreement",
-    "dashboard.agreements.widget.comingSoon": "Coming soon "
+    "dashboard.agreements.widget.comingSoon": "Coming soon",
+    "badge.new": "New",
+    "badge.top": "Top",
+    "badge.dedicated": "Dedicated",
```

Comment:
We already have this exact set of message descriptor ids defined, in this very file: 
https://github.com/JoinColony/colonyCDapp/blob/fdfe45b23021a23ed956bae83ba39d6ce27c7675/src/i18n/en.json#L941-L943

What's the difference between them ?

Reply:
I wanted to make label translations consistent. This `dedicated` translation can be used anywhere and can be changed later to something that won't fit the badge. A lot of translations are related to the component/page that they are used on.